### PR TITLE
loginPage 에러메세지 추가입니다.

### PR DESCRIPTION
**Pull request by chepchep2**
LoginPage 에러메세지 추가입니다.
- 구현영상입니다.
영상이 비밀번호 찾기 영상이랑 같습니다..
 https://imgur.com/KZkhSPH

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/page/login_page.dart [hunk 1]
```diff
@@ -36,10 +39,12 @@ class _LoginPageState extends State<LoginPage> {
     final email = emailController.text;
     if (!validateEmail(email)) {
       setState(() {
+        emailError = AppTextList.emailNotFoundErrorMessage;
         checkEmail = false;
       });
     } else {
       setState(() {
+        emailError = null;
```
Comment (flowkater) [inline lib/page/login_page.dart:47]: 변수에 null을 넣어주는 것은 안티패턴입니다.
왠만하면 사용안하면 좋을 것 같습니다.
value 값으로 분기를 할때는 그냥 boolean 을 쓰는게 좋을 것 같습니다.

File: lib/page/login_page.dart [hunk 2]
```diff
@@ -112,7 +119,14 @@ class _LoginPageState extends State<LoginPage> {
                     onEditingComplete: checkEmailStatus,
                     isEmailValid: checkEmail,
                   ),
-                  const SizedBox(height: 19),
+                  const SizedBox(height: 8),
+                  if (emailError != null)
```
Comment (flowkater) [inline lib/page/login_page.dart:123]: 위 코멘트 참고..

**Review by flowkater — CHANGES_REQUESTED** [review]
참고바랍니다.

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32).
* **Mar 17, 9:02 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32) rebased this pull request as part of a merge.
* **Mar 17, 9:04 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32).

**Pull request merged**